packages/passport/sdk/src/confirmation/confirmation.ts

```diff
@@ -82,8 +82,8 @@ export default class ConfirmationScreen {
       if (chainType === TransactionApprovalRequestChainTypeEnum.Starkex) {
         href = this.getHref('transaction', { transactionId, etherAddress, chainType });
       } else {
-        href = this.getHref('zkevm', {
-          transactionId, etherAddress, chainType, chainId,
+        href = this.getHref('zkevm/transaction', {
+          transactionID: transactionId, etherAddress, chainType, chainID: chainId,
```

Comment:
Does `chainID` need to be uppercase here?

Reply:
Yes, although we have accepted both **ID or Id** on the passport screen. It is better to set to ID as the guardian API needs to be set to **chainID**. And uppercase **ID** seems like a standard in backend API.